rust/crd/src/lib.rs

```diff
@@ -841,6 +843,11 @@ pub struct NameNodeConfig {
     pub logging: Logging<NameNodeContainer>,
     #[fragment_attrs(serde(default))]
     pub affinity: StackableAffinity,
+    #[fragment_attrs(serde(default))]
+    #[fragment_attrs(schemars(
+        description = "Time period Pods have to gracefully shut down, e.g. `30m`, `1h` or `2d`. Consult the operator documentation for details."
+    ))]
```

Comment:
This looks like a fragmentization bug to me, we should probably copy over doccomments into the fragment field (and let schemars use that to generate the description).

Reply:
Created https://github.com/stackabletech/operator-rs/pull/675

Reply:
Merged it and started using it in dcb6bbb9aa18f5477f2d2326e8f333ba6bc173f5